docs/reference/components/catalog.md

```diff
@@ -0,0 +1,366 @@
+```
+
+Run `kubectl describe artifactgenerator generator-9689366613293914683 -n greenhouse` to see the reconciliation status conditions of the `ArtifactGenerator`
+
+```shell
+Status:
+  Conditions:
+    Last Transition Time:  2025-10-31T00:14:59Z
+    Message:               reconciliation succeeded, generated 1 artifact(s)
+    Observed Generation:   2
+    Reason:                Succeeded
+    Status:                True
+    Type:                  Ready
+  Inventory:
+    Digest:                 sha256:a6114ad3b1c591f1585d78818320d603e78d29b04f527c88321027c59372d506
+    Filename:               2528970247.tar.gz
+    Name:                   artifact-9689366613293914683
+    Namespace:              greenhouse
+  Observed Sources Digest:  sha256:bc9221b47aecc3f4c75f41b8657a3a7c985823487da94b8521803251a3628030
+```
+
+If there was an error accumulating the manifests, you will see errors in the `ArtifactGenerator` status conditions. (The same error message and ready status will also be reflected in the `Catalog` `.status.inventory` for the respective source.)
+
+```shell
+  - message: >-
+      artifact-9689366613293914683 build failed: failed to apply copy
+      operations: failed to apply copy operation from
+      '@catalog/thanos/plugindefinition.yamls' to
+      '@artifact/catalogs/thanos/plugindefinition.yamls': source
+      'thanos/plugindefinition.yamls' does not exist
+    observedGeneration: 3
+    reason: BuildFailed
+    status: "False"
+    type: Ready
+```
+
+Finally, the accumulated manifests are applied using Flux `Kustomization`. Run `kubectl get kustomization kustomize-9689366613293914683 -n greenhouse` to see the status.
+
+```shell
+NAME                            AGE     READY   STATUS
+kustomize-9689366613293914683   7d11h   True    Applied revision: latest@sha256:a6114ad3b1c591f1585d78818320d603e78d29b04f527c88321027c59372d506
+```
+
+Run `kubectl describe kustomization kustomize-9689366613293914683 -n greenhouse` to see the reconciliation status conditions of the `Kustomization`
+
+Every organization namespace is equipped with `Role` and `RoleBinding` to a catalog specific `ServiceAccount`.
+The `Role` grants permissions to manage only `PluginDefinition` resources within the organization namespace.
+In case of permissions issues, you will see errors in the `Kustomization` status conditions. (The same error message and ready status will also be reflected in the `Catalog` `.status.inventory` for the respective source.)
+
+```shell
+  - lastTransitionTime: "2025-10-31T00:45:08Z"
+    message: >
+      PluginDefinition/greenhouse/cert-manager dry-run failed
+      (Forbidden): plugindefinitions.greenhouse.sap "cert-manager" is
+      forbidden: User "system:serviceaccount:greenhouse:greenhouse-catalog-sa"
+      cannot patch resource "plugindefinitions" in API group
+      "greenhouse.sap" at the cluster scope
+    observedGeneration: 4
+    reason: ReconciliationFailed
+    status: "False"
+    type: Ready
+```
```

Comment:
This could also go under `## Working with Catalogs` as dedicated sections with practical examples